New components and pages translated.

gongalf opened the pull request:
- The translation system is now integrated into the components and pages contained in the app/map, app/post and app/walkthrough folders.
- The system is now working with the Walkthrough select spelling proposal, Select spelling proposal and Profile screens.

gongalf committed:
Added spelling proposal selection and new translated screens.

The system is now working with the Walkthrough select spelling proposal, Select spelling proposal and Profile screens.

gongalf committed:
New components and pages translated.

The translation system is now integrated into the components and pages contained in the app/map, app/post and app/walkthrough folders.

ssoto commented:
Graçiâ por contribuîh @gongalf 
Êpperamô a que @fontanon le exe un bîttaço pa mêcclâl-lo çi te pareçe.

gongalf committed:
Last components and pages translated.

The translation system is now integrated into the components and pages contained in the app/core, app/shared/components, app/profile/information, app/map/location-select and app/profile/posts.
